components/director/internal/domain/formation/assign_formation_test.go

```diff
@@ -1374,9 +1458,11 @@ func TestServiceAssignFormation(t *testing.T) {
 			FormationTemplateRepositoryFn: func() *automock.FormationTemplateRepository {
 				repo := &automock.FormationTemplateRepository{}
 				repo.On("Get", ctx, FormationTemplateID).Return(&model.FormationTemplate{
-					ID:           FormationTemplateID,
-					Name:         "some-other-template",
-					RuntimeTypes: []string{"not-the-expected-type"},
+					ID:                     FormationTemplateID,
+					Name:                   "some-other-template",
+					RuntimeArtifactKind:    &subscriptionRuntimeArtifactKind,
+					RuntimeTypeDisplayName: str.Ptr("display name"),
```

Comment:
In a couple of places, we use that `str.Ptr("display name"),` we could extract it as a variable in the fixtures and reuse it here?